common/persistence/dataInterfaces.go

```diff
@@ -197,6 +198,7 @@ type (
 		ShardID          int32
 		CreateIfMissing  bool
```

Comment:
why there is a `create if missing?`

Reply:
just to avoid changing semantics. removed.